styleguide/source/_patterns/02-organisms/topic/topic-rail-right/topic-rail-right.twig

```diff
@@ -0,0 +1,3 @@
+<aside class="topic_rail-right">
```

Comment:
Ok also lmk if we should remove because I also added that concept to https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/169 after seeing Rachel's.